<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWRunAroundProperties</class>
 <widget class="QWidget" name="KWRunAroundProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>456</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Text elsewhere will:</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="runThrough">
        <property name="text">
         <string>Run through this shape</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="noRunaround">
        <property name="text">
         <string>Skip below this shape</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="left">
        <property name="text">
         <string>Run on left side of this shape</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="right">
        <property name="text">
         <string>Run on right side of this shape</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="longest">
        <property name="text">
         <string>Run on longest side of this shape</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="both">
        <property name="text">
         <string>Run on both sides of this shape</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="enough">
          <property name="text">
           <string>Run on sides that are wider than:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KoUnitDoubleSpinBox" name="threshold">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>The shape affects the text by its:</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="box">
        <property name="text">
         <string>Bounding box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="outside">
        <property name="text">
         <string>Contour</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Shape and text is separated:</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Left by:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>distanceLeft</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KoUnitDoubleSpinBox" name="distanceLeft"/>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>180</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Above by:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>distanceLeft</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KoUnitDoubleSpinBox" name="distanceTop"/>
      </item>
      <item row="2" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>180</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Below by:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>distanceLeft</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KoUnitDoubleSpinBox" name="distanceBottom"/>
      </item>
      <item row="4" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>180</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Right by:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>distanceLeft</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KoUnitDoubleSpinBox" name="distanceRight"/>
      </item>
      <item row="1" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>180</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoUnitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>KoUnitDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
